### PR TITLE
India - Periodic Labour Force Survey (PLFS) -  Average daily wage earnings

### PR DESCRIPTION
**Pull request by thejeshgn**
- Added 2017-2018 data
- Added 2018-2019 data

### CODE REVIEW

File: scripts/india_plfs/daily_wage_data/PLFSDailyWageData_India_StatisticalVariables.mcf [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Node: dcid:Mean_WagesDaily_Worker_Rural_Male
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+statType: dcs:meanValue
+measuredProperty: dcs:WagesDaily
```
Comment (pradh) [inline scripts/india_plfs/daily_wage_data/PLFSDailyWageData_India_StatisticalVariables.mcf:5]: wagesDaily

(everywhere else too)

File: scripts/india_plfs/daily_wage_data/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Periodic Labour Force Survey (PLFS) -  Average daily wage earnings
+
+
+## About the Dataset
+Average wage earnings (Rs. 0.00) per day from casual labour work other than public works in CWS for each State/ UT.
```
Comment (pradh) [inline scripts/india_plfs/daily_wage_data/README.md:5]: nit: expand CWS or link to what it means?

File: scripts/india_plfs/daily_wage_data/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+### Download Data
+
+#### PLES July 2018 - June 2019
+Wage data from PLES July 2018 - June 2019 is available as [XLSX](http://mospi.nic.in/sites/default/files/reports_and_publication/PLFS_2018_19_Anual/Table_43.xlsx).
+
+#### PLES July 2017 - June 2018
+Wage data from PLES July 2017 - June 2018 is available as embedded tables inside the  [PDF](http://mospi.nic.in/sites/default/files/publication_reports/Annual%20Report%2C%20PLFS%202017-18_31052019.pdf). Table format is the same. Extracted as XLSX using [tabula](https://github.com/tabulapdf/tabula) by running the command
+
+```
+java -jar tabula-java.jar  -a 40.506,97.049,521.453,782.161 -p 404 "$1"
```
Comment (pradh) [inline scripts/india_plfs/daily_wage_data/README.md:16]: for my understanding, what does -a argument do?

File: scripts/india_plfs/daily_wage_data/preprocess_test.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+            loader = PLFSDailyWageDataLoader(xlsx_file, period="2018-10")
+            loader.load()
+            loader.process()
+            loader.save(csv_file_path=result_file)
+
+            same = filecmp.cmp(result_file, expected_file)
+            os.remove(result_file)
+
+        self.assertTrue(same)
```
Comment (pradh) [inline scripts/india_plfs/daily_wage_data/preprocess_test.py:45]: nit: can we do self.assertEqual() instead to see diffs when test fails?

**Review by pradh — COMMENTED** [review]
Nice!

File: scripts/india_plfs/daily_wage_data/README.md (hunk 2 above)
Comment (thejeshgn) [reply scripts/india_plfs/daily_wage_data/README.md:5]: It's already there in the **Notes from the data page :** section.

File: scripts/india_plfs/daily_wage_data/README.md (hunk 3 above)
Comment (thejeshgn) [reply scripts/india_plfs/daily_wage_data/README.md:16]: `-a/--area = Portion of the page to analyze.`

File: scripts/india_plfs/daily_wage_data/README.md (hunk 2 above)
Comment (thejeshgn) [reply scripts/india_plfs/daily_wage_data/README.md:5]: **Notes from the data page :** 
1. The Current Weekly Status (CWS) approach to measuring uses seven days preceding the date of survey as the reference period.

**Review by pradh — APPROVED** [review]
Thanks!

**Pull request merged**